packages/block-library/src/query-pagination-numbers/edit.js

```diff
@@ -19,7 +19,13 @@ const previewPaginationNumbers = () => (
 	</>
 );
 
-export default function QueryPaginationNumbersEdit() {
+export default function QueryPaginationNumbersEdit( attributes ) {
+	const blockProps = useBlockProps();
+	const blockGap = attributes.attributes?.style?.spacing?.blockGap;
 	const paginationNumbers = previewPaginationNumbers();
-	return <div { ...useBlockProps() }>{ paginationNumbers }</div>;
+	return (
+		<div { ...blockProps } style={ { ...blockProps.style, gap: blockGap } }>
```

Comment:
That threw me off a bit... It seems that `blockGap support` is used only in `layout` abstraction and that's why this code is needed. Do you think this would be the proper handling of such cases? --cc @ramonjd @andrewserong @aaronrobertshaw @youknowriad